Add Neumann

imstar15 opened the pull request:
![image](https://user-images.githubusercontent.com/16951509/148101154-5992de8c-4ff7-43a2-90f4-9698935caf3f.png)

![image](https://user-images.githubusercontent.com/16951509/148101197-185ca50f-d148-44e9-94ee-9c9acb42c151.png)

imstar15 committed:
Add Neumann

irsal commented on packages/apps-config/src/endpoints/testing.ts:
```diff
@@ -316,10 +316,10 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
+      text: t('rpc.test.oak', 'Neumann Network', { ns: 'apps-config' }),
       providers: {
-        'OAK Network': 'wss://rpc.testnet.oak.tech'
+        'OAK Network': 'wss://node-6874600165100077056.jm.onfinality.io/ws?apikey=00e65cb5-0bcc-4dac-9a6d-08e6e111e186'
```
we need to switch this to the api service onfinality provides, once we have a version we're comfortable with, then we can proceed

chrisli30 replied:
We need to bind the actual API endpoint to the domain `rpc.testnet.oak.tech`, rather than exposing it, so this line of code shouldn’t be changed.

chrisli30 replied:
Another reason is that the domain rpc.testnet.oak.tech is easier to remember.

irsal replied:
`wss://oak.api.onfinality.io/public-ws` We can get the api service from onfinality and bind it to that. We can add the rpc bind at a later time.

chrisli30 replied:
Sounds good!

imstar15 replied:
How does it distinguish networks (OAK, Turing, Neumann) by one URL?

`wss://oak.api.onfinality.io/public-ws`

@irsal

jacogr committed:
2022 (#6764)

actions-user committed:
[CI Skip] release/beta 0.100.2-16

skip-checks: true

jacogr committed:
Bump dev (#6765)

* Bump dev

* Rebuild

actions-user committed:
[CI Skip] release/beta 0.100.2-17

skip-checks: true

jacogr committed:
Cleanup subsocial import (dedupe package error) (#6766)

actions-user committed:
[CI Skip] release/beta 0.100.2-18

skip-checks: true

jacogr committed:
2022 (#6769)

actions-user committed:
[CI Skip] release/beta 0.100.2-19

skip-checks: true

jacogr committed:
Adjust tsconfig types location (#6771)

actions-user committed:
[CI Skip] release/beta 0.100.2-20

skip-checks: true

jacogr committed:
Bump dev (w/ header fixes) (#6772)

actions-user committed:
[CI Skip] release/beta 0.100.2-21

skip-checks: true

chrisli30 commented on packages/apps-config/src/ui/colors.ts:
```diff
@@ -58,7 +58,7 @@ const chainKulupu = '#003366';
 const chainMoonrock = '#3d1d5a';
 const chainNFTMart = '#815287';
-const chainOakTestnet = '#E38985';
+const chainNeumann = '#E38985';
```
Change all colors `#E38985` to `#A8278C` as that’s our new logo color

chrisli30 commented on packages/apps-config/src/endpoints/testing.ts:
```diff
@@ -316,10 +316,10 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
     {
-      info: 'oak-testnet',
-      text: t('rpc.test.oak', 'OAK Testnet', { ns: 'apps-config' }),
+      info: 'neumann',
```
When we register to polkadot ss58, we register the prefix for Turing Network. 

Neumann as a test can use the same prefix. This is what Astar Network did, using the same prefix as registered ones.

Let’s use the prefix 27 for "Turing Network", as that’s the first available in the front (from 0).

chrisli30 commented on packages/apps-config/src/ui/logos/index.ts:
```diff
@@ -110,9 +110,9 @@ import moonriver from './nodes/moonriver.svg';
 import nodeMoonrock from './nodes/moonrock.png';
 import moonshadow from './nodes/moonshadow.png';
 import mybank from './nodes/mybank.png';
+import neumann from './nodes/neumann.png';
```
icon should be updated too. Use the below new icon.

https://drive.google.com/file/d/1fYML6mTEnQJNiG_R3NGzfv6dQfLLvobM/view?usp=sharing

jacogr committed:
fetch-depth 0 (#6776)

wbh1328551759 committed:
Add Sherpax Testnet node (#6775)

* Add Sherpax Testnet node

* fix lint

* fix test in testing

* move sherpax node into live networks

* add color of sherpax testnet

* move sherpax node behind Robonomics

* Add logo of Sherpax

* fix lint

actions-user committed:
[CI Skip] release/beta 0.100.2-22

skip-checks: true

zzcwoshizz committed:
rm elara endpoint (#6778)

actions-user committed:
[CI Skip] release/beta 0.100.2-23

skip-checks: true

gregdhill committed:
add custom derive for interbtc native balances (#6779)

Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

imstar15 committed:
Modify the logo and color for neumann.

actions-user committed:
[CI Skip] release/beta 0.100.2-24

skip-checks: true

chrisli30 commented on packages/apps-config/src/ui/colors.ts:
```diff
@@ -156,9 +156,9 @@ const nodeMathChain = '#000000';
 const nodeMoonbeam = '#53cbc9';
 const nodeMoonriver = '#0E132E';
 const nodeMybank = '#282736';
+const nodeNeumann = '#E38985';
```
const nodeNeumann = '#E38985';

This needs to be changed to the new color too. Please review your own PR before submit it.

imstar15 committed:
Delete nodeNeumann in packages/apps-config/src/ui/colors.ts

imstar15 committed:
Modify the endpoint of neumann.

imstar15 committed:
Change order of colors and logos list.

DrW3RK committed:
Clarify Teleport transaction fees message (#6781)

As the destination transfer fee estimate is removed in https://github.com/polkadot-js/apps/pull/6745/ , it is resourceful to update the message mentioning that the teleported amount is subject to destination transfer fees along with the transfer fees at the source (and the balance should be above Existential Deposit, if the recipient's account is new

actions-user committed:
[CI Skip] release/beta 0.100.2-25

skip-checks: true

chrisli30 reviewed (approved):
Looks good to me. Wait to test `'wss://neumann.api.onfinality.io/public-ws'` before merging this to upstream.

jacogr committed:
Bump dev (#6786)

shenger9 committed:
update-mandala (#6785)

actions-user committed:
[CI Skip] release/beta 0.100.2-26

skip-checks: true

gregdhill committed:
don't crash transfer model if balances is not available (#6787)

Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

actions-user committed:
[CI Skip] release/beta 0.100.2-27

skip-checks: true

jacogr committed:
Bump dev (w/ detectPackage) (#6791)

* Bump dev (w/ detectPackage)

* Additional

actions-user committed:
[CI Skip] release/beta 0.100.2-28

skip-checks: true

simonsso committed:
Polkadex testnet have been resurrected (#6783)

Polkadex testnet is running again.

actions-user committed:
[CI Skip] release/beta 0.100.2-29

skip-checks: true

sxlwar committed:
crab parachain (#6792)

* crab parachain

* crab plo

* order

actions-user committed:
[CI Skip] release/beta 0.100.2-30

skip-checks: true

jacogr committed:
Crab isUnreachable (#6795)

actions-user committed:
[CI Skip] release/beta 0.100.2-31

skip-checks: true

chrisli30 reviewed (commented):
I’m working with our logo designer for Turing’s logo so wait for that.

irsal commented:
turing? this is for neumann. What's the eta on that, Chris?

Irsal Alsanea
Co-founder, OAK Network
Telegram: @irsalj
Website: https://oak.tech


On Thu, Jan 06, 2022 at 4:32 PM, Chris Li ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> I’m working with our logo designer for Turing’s logo so wait for that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OAK-Foundation/apps/pull/3#pullrequestreview-846188372>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAM5QXPA37KMSZUBGQ6P42LUUYYCTANCNFSM5LBGNB6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

imstar15 committed:
Merge branch 'master-polkadot-js' into add_neumann

imstar15 committed:
Put neumann in order in src/endpoints/testing.ts
